qa/vagrant/src/test/resources/packaging/utils/utils.bash

```diff
@@ -415,6 +415,50 @@ stop_elasticsearch_service() {
     fi
 }
 
+# the default netcat packages in the distributions we test are not all compatible
```

Comment:
You might want to expand this comment a bit with something like "so we have to use /dev/tcp" or something.